scripts/install

```diff
@@ -61,31 +68,38 @@ ynh_exec_fully_quiet myynh_install_homeassistant --path="$data_path"
 # set default configuration files and move all homeassistant_conf_files
 ynh_script_progression --message="Configuring the installation..."
 ynh_replace_string --match_string="__PORT__" --replace_string="$port" --target_file="../conf/homeassistant_conf_files/configuration.yaml"
-ynh_replace_string --match_string="__PATH__" --replace_string="$data_path/.$app" --target_file="../conf/homeassistant_conf_files/configuration.yaml"
+ynh_replace_string --match_string="__APP__" --replace_string="$app" --target_file="../conf/homeassistant_conf_files/configuration.yaml"
+ynh_replace_string --match_string="__DATAPATH__" --replace_string="$data_path" --target_file="../conf/homeassistant_conf_files/configuration.yaml"
 cp -r "../conf/homeassistant_conf_files/." "$data_path/.$app/"
```

Comment:
Sounds like you could really simplify this with `ynh_add_config`

Reply:
seems that `ynh_add_config` only wor kwith file but i need to move a whole directory

Reply:
Alex was referring to the multiple `ynh_replace_string` commands. ;)

Reply:
hum i not sur to understand your advice, is it that ?
`cp -r "../conf/homeassistant_conf_files/." "$data_path/.$app/"`
`ynh_add_config --template="$data_path/.$app/configuration.yaml" --destination="$data_path/.$app/configuration.yaml"`

Reply:
Yes, or using `./conf/homeassistant_conf_files/configuration.yaml` as `--template` in the second command (same result, but it's better from a semantic point of view)

(As a side note I'm super confused that the conf happens to be in `$data_path/.$app` x_x, I'm not sure if this is a deliberate choice or just the app internal structure imposing this)

Reply:
ok but it seems i need to keep 2 ynh_replace_string :
`cp -r "../conf/homeassistant_conf_files/." "$data_path/.$app/"`
`ynh_replace_string --match_string="__PORT__" --replace_string="$port" --target_file="../conf/homeassistant_conf_files/configuration.yaml"`
`ynh_replace_string --match_string="__DATAPATH__" --replace_string="$data_path" --target_file="../conf/homeassistant_conf_files/configuration.yaml"`
`ynh_add_config --template="../conf/homeassistant_conf_files/configuration.yaml" --destination="$data_path/.$app/configuration.yaml"`
is that fine ?
Regarding `$data_path/.$app` it's the default app structure (never try too move it but can try)

Reply:
Why would you need to keep those 2 replace_string ? `ynh_add_config` will take care of replacing `__PORT__` with `$port` ... and should be replaced `__DATAPATH__` in the template by `__DATA_PATH__` (note the middle `_`) to be replaced by `$data_path`